crates/router/src/core/payment_methods/cards.rs

```diff
@@ -3738,7 +3732,7 @@ pub async fn call_surcharge_decision_management_for_saved_card(
         .attach_printable("Could not decode the routing algorithm")?
         .unwrap_or_default();
     // TODO: Move to business profile surcharge column
```

Comment:
Nit: Could you move this comment down? I believe this was meant to be above the `let surcharge_results = ...` line.